src/algokit/cli/init.py

```diff
@@ -61,14 +81,17 @@ def _get_blessed_templates() -> dict[str, BlessedTemplateSource]:
         "puya": BlessedTemplateSource(
             url="gh:algorandfoundation/algokit-puya-template",
             description="Official starter template for Puya applications (Dev Preview, not recommended for production)",
+            branch="poc/wizard_v2",
```

Comment:
@robdmoore @neilcampbell please note this is a temporary change done for the sake of adr given that dependant templates also rely on changes on specific branch